packages/flutter/lib/src/material/page.dart

```diff
@@ -63,13 +63,14 @@ class MaterialPageRoute<T> extends PageRoute<T> with MaterialRouteTransitionMixi
 /// A mixin that provides platform-adaptive transitions for a [PageRoute].
 ///
 /// {@template flutter.material.materialRouteTransitionMixin}
-/// For Android, the entrance transition for the page slides the route upwards
-/// and fades it in. The exit transition is the same, but in reverse.
-///
-/// The transition is adaptive to the platform and on iOS, the route slides in
-/// from the right and exits in reverse. The route also shifts to the left in
-/// parallax when another page enters to cover it. (These directions are flipped
-/// in environments with a right-to-left reading direction.)
+/// For Android, the entrance transition for the page zooms in while the
+/// exiting page zooms and fades out. The exit transition is similar, but in
+/// reverse.
```

Comment:
I see that the `_ZoomExitTransition` and `_ZoomEnterTransition` both have a `FadeTransition` and a `ScaleTransition`—would it be more accurate to say that the entrance `zooms in and fades in while the exiting page zooms out and fades out`?

Reply:
Sounds better, I'll leave this until the whole thing looks good.